src/scripts/main.js

```diff
@@ -1,7 +1,112 @@
 'use strict';
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+const Game = require('../modules/Game.class');
```

Comment:
The use of `require` suggests a Node.js environment. If this code is intended to run in a browser with ES6 modules, consider using `import Game from '../modules/Game.class';` instead.